save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/BenchmarkEntity.kt

```diff
@@ -27,7 +30,7 @@ enum class BenchmarkCategoryEnum {
         private val postfixInRegex = values().joinToString("|") { it.name.lowercase() }
         override val nameOfTheHeadUrlSection = "archive"
         override val defaultTab: BenchmarkCategoryEnum = ALL
-        override val regexForUrlClassification: Regex = Regex("/$nameOfTheHeadUrlSection/[^/]+/($postfixInRegex)")
```

Comment:
It will be cast/ or used as regex under hood?

Reply:
yes, when it's needed, regex will be created inside a method. Regex cannot be a part of public api